Tools/demo/vector.py

```diff
@@ -50,22 +56,28 @@ def __getitem__(self, i):
         return self.v[i]
 
     def __add__(self, other):
-        # Element-wise addition
+        "Element-wise addition"
```

Comment:
Isn't best practise triple quotes for docstrings? Ref. [PEP 8/documentation-strings](https://www.python.org/dev/peps/pep-0008/#documentation-strings)

Reply:
We're somewhat loose about this.  The original rationale was that it would save you from retyping the quotes if the docstring grew to multiple lines. That isn't a likely possibility here.